src/Symfony/Component/Yaml/README.md

```diff
@@ -1,21 +1,33 @@
 Yaml Component
 ==============
 
-YAML implements most of the YAML 1.2 specification.
+[![Package version](https://img.shields.io/packagist/v/symfony/yaml.svg)](https://packagist.org/packages/symfony/yaml)
```

Comment:
well, you generally don't need to know it as ``composer require symfony/yaml`` will guess a constraint based on the available versions